### PR TITLE
Update blocks with latest middleware changes

### PR DESCRIPTION
**Pull request by sayedtaqui**

### CODE REVIEW

File: blocks/timeline/index.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ registerBlockType( 'rtgb/timeline', {
 				className: 'timeline-title',
 				placeholder: __( 'Enter title' ),
 				tagName: 'h4',
+				formattingControls: [ '' ],
```
Comment (sayedtaqui) [inline blocks/timeline/index.js]: Is it because we do not want to show toolbar?

Comment (manishsongirkar) [reply blocks/timeline/index.js]: Yes, If we use `inlineToolbar: false` then it will stick to block level. But no option to completely remove it.

Comment (sayedtaqui) [reply blocks/timeline/index.js]: Then I think it should be `formattingControls: []` , in place of `formattingControls: [ '' ]`

Comment (manishsongirkar) [reply blocks/timeline/index.js]: Updated.

**Pull request merged**